lines.js

```diff
@@ -94,5 +136,40 @@ function chooseSize(e) {
     dotSize = parentSquare;
     dotSize.classList.add("chosen");
     radius = dotSize.dataset.size;
+    context.lineWidth = 2 * radius;
 }
 
+function turnBack(e) {
+    if (currentIndex !== 0) {
+        context.clearRect(0, 0, myCanvas.width, myCanvas.height);
+        currentIndex -= 1;
+        context.drawImage(projectHistory[currentIndex], 0, 0);
+    }
+}
+
+function turnForward(e) {
+    if (currentIndex !== historyChanges) {
+        context.clearRect(0, 0, myCanvas.width, myCanvas.height);
+        currentIndex += 1;
+        context.drawImage(projectHistory[currentIndex], 0, 0);
+    }
+}
+
+function drawPoint(r, x, y) {
+    context.fillRect(x - 1, y - 1, 2, 2);
+    if (r > 1) {
+        context.fillRect(x - r, y - 1, 2 * r, 2);
+        context.fillRect(x - 1, y - r, 2, 2 * r);
+        if (r === "3") {
```

Comment:
Proper way is to realize which type `r` has and cast to that type. 
Currently you expect `r: number` at `r > 1`, but `r: string` at ` === "3"`

Reply:
Can not we cast the variable at 1st line and use number further?

Reply:
The comment still relevant